File: app/src/main/java/com/zeros/modulelibads/MainActivity.java
package com.zeros.modulelibads;

import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.Button;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

import com.zeros.ads.ads.AmxAd;
import com.zeros.ads.ads.wrapper.ApInterstitialAd;
import com.zeros.ads.funtion.AdCallback;


public class MainActivity extends AppCompatActivity {
    private ApInterstitialAd mInterstitialAd;
    private Button btnShow;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        AmxAd.getInstance().getInterstitialAds(this, BuildConfig.ad_interstitial_splash, new AdCallback(){
            @Override
            public void onApInterstitialLoad(@Nullable ApInterstitialAd apInterstitialAd) {
                super.onApInterstitialLoad(apInterstitialAd);
                mInterstitialAd = apInterstitialAd;
            }
        });

        btnShow = findViewById(R.id.btnShow);
        btnShow.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                AmxAd.getInstance().forceShowInterstitial(MainActivity.this, mInterstitialAd, new AdCallback(){

                }, false);
            }
        });

    }
}

File: app/src/main/java/com/zeros/modulelibads/MainActivity.kt
package com.zeros.modulelibads

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.zeros.ads.admob.Admob
import com.zeros.ads.funtion.AdCallback

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        Admob.getInstance().loadSplashInterstitialAds(
            this,
            BuildConfig.ad_interstitial_splash,
            25000,
            5000,
            object : AdCallback() {
                override fun onNextAction() {
                    super.onNextAction()

                }
            })
    }
}